srv/salt/ceph/macros/os_switch.sls

```diff
@@ -5,12 +5,20 @@
   {%- set osrelease = os + "-" + grains.get('osrelease') -%}
   {%- if salt['file.directory_exists'](abspath + "/" +  custom) -%}
     {{ custom }}
+  {%- elif salt['file.file_exists'](abspath + "/" +  custom + ".sls") -%}
```

Comment:
I'm aware that a 'path' should not end with a '/' but in this case we are not using it as a standalone variable which makes all if-clauses a bit redundant:

`abspath + "/" +  custom + ".sls"`

vs
` {% set abspath = "/srv/salt/" + slspath + '/' %}`

and then 

`abspath +  custom + ".sls"`

not sure if I like it either :)

Reply:
:+1: